vision/cloud-client/detect/beta_snippets.py

```diff
@@ -174,6 +174,99 @@ def detect_handwritten_ocr_uri(uri):
 # [END vision_handwritten_ocr_gcs_beta]
 
 
+# [START vision_fulltext_detection_pdf_beta]
+def detect_document_features(path):
+    """Detects document features in a PDF/TIFF/GIF file.
+
+    Args:
+    path: The path to the local file.
+    """
+    from google.cloud import vision_v1p4beta1 as vision
+    client = vision.ImageAnnotatorClient()
+
+    with open(path, 'rb') as pdf_file:
+        content = pdf_file.read()
+
+    # Other supported mime_types: image/tiff' or 'image/gif'
+    mime_type = 'application/pdf'
+    input_config = vision.types.InputConfig(
+        content=content, mime_type=mime_type)
+
+    feature = vision.types.Feature(
+        type=vision.enums.Feature.Type.DOCUMENT_TEXT_DETECTION)
+    # Annotate the first two pages and the last one (max 5 pages)
+    # First page starts at 1, and not 0. Last page is -1.
+    pages = [1, 2, -1]
```

Comment:
nit: Since I was confused the first time could add like:
```
pages = [1, 2, -1]  # Syntax for a 3 page pdf file
```

Reply:
To clarify: your PDF can have many pages, but you get to choose up to 5 pages of it to be processed.

Reply:
Oh... interesting. That itself might be worth commenting in the docs.

Reply:
I appreciate having this as part of the example, with the comment!

As a user, I may not have stumbled upon this otherwise :)